### DISCUSSION TITLE
Backup settings

### DISCUSSION THREAD

**Discussion by vanbang9710** [WindowTop/WindowTop-App#335] [General]
Is the settings.json file enough? ![|50pct](https://i.imgur.com/e4JWqhn.png)

**Comment by gileli121** [comment]
Yes, 
`settings.json` stores everything, including the activation key.
Please note that if you distribute the app with settings to everyone, deactivate the app by clicking on the "Deactivate" button
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/e6ed3a82-fb9b-4536-93d6-2afaacf2637d)
